.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: ruby
+rvm:
+  - 2.2.0
```

Comment:
Why 2.2 ? can't you use 2.5.3 or higher ?

Reply:
because... :sad-smile-oldman:
Done!.